source/viz.md

```diff
@@ -255,32 +256,38 @@ Scatter plots show the data as individual points with `x` (horizontal axis)
 Here, we will use the measurement date as the `x` coordinate
 and the CO$_{\text{2}}$ concentration as the `y` coordinate.
-We create a plot object with the `alt.Chart()` function.
+We create a chart with the `alt.Chart()` function.
 There are a few basic aspects of a plot that we need to specify:
 
-```{index} altair; geometric object, altair; geometric encoding, geometric object, geometric encoding
+```{index} altair; graphical mark, altair; encoding channel
 ```
 
-- The name of the **data frame** object to visualize.
+- The name of the **data frame** to visualize.
     - Here, we specify the `co2_df` data frame as an argument to `alt.Chart`
-- The **geometric object**, which specifies how the mapped data should be displayed.
-	- To create a geometric object, we use `Chart.mark_*` methods (see the
-	  [altair reference](https://altair-viz.github.io/user_guide/marks.html)
-	  for a list of geometric objects).
+- The **graphical mark**, which specifies how the mapped data should be displayed.
+    - To create a graphical mark, we use `Chart.mark_*` methods (see the
+      [altair reference](https://altair-viz.github.io/user_guide/marks.html)
+      for a list of graphical mark).
     - Here, we use the `mark_point` function to visualize our data as a scatter plot.
-- The **geometric encoding**, which tells `altair` how the columns in the data frame map to properties of the visualization.
+- The **encoding channels**, which tells `altair` how the columns in the data frame map to visual properties in the chart.
     - To create an encoding, we use the `encode` function.
     - The `encode` method builds a key-value mapping between encoding channels (such as x, y) to fields in the dataset, accessed by field name (column names)
-	- Here, we set the `x` axis of the plot to the `date_measured` variable,
-	  and on the `y` axis, we plot the `ppm` variable. We use `alt.X` and
-	  `alt.Y` which allow you to control properties of the `x` and `y` axes.
-	- For the y-axis, we also provided the argument
-	  `scale=alt.Scale(zero=False)`. By default, `altair` chooses the y-limits
-	  based on the data and will keep `y=0` in view. That would make it
-	  difficult to see any trends in our data since the smallest value is >300
-	  ppm. So by providing `scale=alt.Scale(zero=False)`, we tell altair to
-	  choose a reasonable lower bound based on our data, and that lower bound
-	  doesn't have to be zero.
+    - Here, we set the `x` axis of the plot to the `date_measured` variable,
+      and on the `y` axis, we plot the `ppm` variable.
+    - For the y-axis, we also provided the argument
+      `scale=alt.Scale(zero=False)`. By default, `altair` chooses the y-limits
+      based on the data and will keep `y=0` in view.
+      This is often a helpful default, but here it makes it
+      difficult to see any trends in our data since the smallest value is >300
+      ppm. So by providing `scale=alt.Scale(zero=False)`, we tell altair to
```

Comment:
is "altair" typeset using code or text? `altair` vs altair. We should be consistent throughout. I believe in the R book, packages are typeset using code syntax, so here we would prefer `altair`, if I recall

Reply:
(this will inevitably be a formatting pass that we have to do later on though)

Reply:
In the docs, we just call it "Altair" without typesetting as code and with a leading capital. I agree we should change it in a later formatting pass if at all.

Reply:
We have a style guide for the R textbook here: https://github.com/ubc-dsci/introduction-to-datascience#style-guide

we can just follow this (and update it as necessary) once we are at the level of "final polish" for the official publication